### ISSUE TITLE
Is WifiWizard2 still maintained?

### ISSUE THREAD

**Issue by bendoerr** [tripflex/WifiWizard2#111]
Hello @tripflex. Is this project still maintained?

**Comment by bendoerr** [comment]
@arsenal942 has confirmed this repo is no longer maintained. https://github.com/tripflex/WifiWizard2/issues/99#issuecomment-688836731

If anyone has the wherewithal to pick this up, myself and @Rytiggy may be able to help support you.

**Comment by tripflex** [comment]
Yeah sorry guys, things have been crazy busy for me lately, and i'm no longer working on the project that I was using this library for.  

I have absolutely no problem working with anybody who wants to pick this up and maintain it.  Which is exactly what happened with the original WifiWizard lib, and is why I forked it and created a new one originally.

If you're unable to get a response from me on here join my Zulip at just shoot me a PM (i get tons of emails so i miss a lot of GitHub notifications):  https://chat.smyl.es

**Comment by tripflex** [comment]
@bendoerr just in case :)

**Comment by bendoerr** [comment]
Thanks @tripflex ! My bud @rytiggy is already on top of things.

**Comment by tripflex** [comment]
Okay sounds good, if he's interested in becoming a maintainer let me know, I just don't have the time to work on this anymore and to be honest, I kind of knew this would eventually happen, which is why I tried to document it as much as I could but I think now is time to pass the torch on to someone else.  Mainly because testing this lib is very cumbersome :P

**Comment by josephnaber** [comment]
If anyone is maintaining this or has a branch that works with iOS 14.5 I would like to get in touch
